.gitignore

```diff
@@ -0,0 +1,23 @@
+# Compiled class file
```

Comment:
Please update it with:
- .project,
- .classpath,
- .settings folder

If later you build project with Maven
 then you have to add target as well 
to avoid storing built results.